test/node/src/select.test.ts

```diff
@@ -814,5 +814,28 @@ for (const dialect of BUILT_IN_DIALECTS) {
       const endTime = Date.now()
       console.log((endTime - time) / N)
     })
+
+    it('should accept a conditional function for $.if', async () => {
+      const params: {
+        name?: string
+      } = {}
+
+      params.name = 'Jennifer'
```

Comment:
Is there a good reason for typing this const and splitting assignment to 2 steps.

```suggestion
      const params = { name: 'Jennifer' }; 
```

Reply:
Only to stress that it was set after being defined, but will update to set at `const` instead:

```ts
 const params: {
        name?: string
      } = { name: 'Jennifer' }
```